content/tutorials/r_renv_update/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Updating the renv environment of an R project"
+description: "Updating R and packages in an renv environment"
+authors: [florisvdh]
+date: 2021-06-09
+categories: ["r"]
+tags: ["renv", "r", "packages"]
+---
+
+## What is `renv`?
+
+`renv` is an R package that provides tools to save the version of R and all packages used in a particular directory (e.g. an RStudio project directory), and to reproduce (recreate) this environment on another (or your future) system.
+Because software versions can have important effects on the results of your work, it is important to be able to reproduce versions.
+
+For basic advice on how to setup and use `renv`, see <https://rstudio.github.io/renv>.
```

Comment:
Maybe highlight this section by adding a header. So people can easily find instructions on how to start with `renv`

Reply:
:+1:

Reply:
0e276773